cli/cmd/alert_profiles.go

```diff
@@ -425,6 +425,21 @@ func promptAddAlertTemplate() api.AlertTemplate {
 	}
 }
 
+func filterAlertProfilesByDefault(response api.AlertProfilesResponse) []string {
```

Comment:
`ByNonDefault` makes a bit more sense to me as we are filtering out the Default Profiles
```suggestion
func filterAlertProfilesByNonDefault(response api.AlertProfilesResponse) []string {
```